test/acceptance/pages/UnblockApplicationSuccessPage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.pages
+
+import acceptance.WebPage
+
+object UnblockApplicationSuccessPage extends WebPage {
+
+  override val url: String = s"http://localhost:$port/api-gatekeeper/applications/fa38d130-7c8e-47d8-abc0-0374c7f73217/unblock"
+
+  override def isCurrentPage: Boolean = {
+    currentUrl == url
```

Comment:
This seems to be the same page as the UnblockApplicationPage. Why not use that one?

Reply:
Agreed.

Reply:
However, having made the change the test reads really badly. You get the impression that you haven't changed pages at all :(

I'll leave it in for now even though it appears odd. Makes the test read much better.